### PR TITLE
Cycles with a duration of zero still show misleading information on the upcoming cycle tab.

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

Closes [JB-738 : Upcoming cycle redesign + fix](https://linear.app/peel/issue/JB-738/upcoming-cycle-redesign-fix)

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] (if relevant) I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] (if relevant) I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #4031** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4031" target="_blank">Graphite</a> 
    * **PR #4032** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4032" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4032?utm_source=stack-comment).

File: src/components/ProjectDashboard/components/CyclesPayoutsPanel/components/CurrentUpcomingSubPanel.tsx [hunk 1]
```diff
@@ -62,6 +63,22 @@ export const CurrentUpcomingSubPanel = ({
     </Trans>
   )
 
+  if (info.type === 'upcoming' && info.cycleUnlocked) {
```
Comment (johnnyd-eth) [inline src/components/ProjectDashboard/components/CyclesPayoutsPanel/components/CurrentUpcomingSubPanel.tsx:66]: factsss

**Review by johnnyd-eth — APPROVED** [review]
LGTM

**Comment by wraeth-eth** [comment]
### Merge Activity

* **Aug 29, 5:26 PM**: @wraeth-eth started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4032).

**Pull request merged**